app/scripts/controllers/preferences.js

```diff
@@ -70,6 +70,7 @@ export default class PreferencesController {
         : LedgerTransportTypes.u2f,
       transactionSecurityCheckEnabled: false,
       theme: ThemeType.os,
+      openSeaTransactionSecurityProviderPopoverHasBeenShown: false,
```

Comment:
[suggestion] for boolean names there are suggested prefixes:  "is", "has", "should", "can", "did", "will" (pulled from ESLint rule we are not using). It can help with readability by understanding the intent of the return value reading left to right.

we could update `openSeaTransactionSecurityProviderPopoverHasBeenShown` -> `hasShownOpenSeaTransactionSecurityProviderPopover`

ref ESLint rule: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-boolean-variables-are-prefixed-with-an-allowed-verb